src/Core/System/SalesChannel/Subscriber/SalesChannelAnalyticsLoader.php

```diff
@@ -32,8 +35,7 @@ public function loadAnalytics(StorefrontRenderEvent $event): void
         $criteria = new Criteria([$analyticsId]);
         $criteria->setTitle('sales-channel::load-analytics');
 
-        /** @var SalesChannelAnalyticsEntity|null $analytics */
-        $analytics = $this->salesChannelAnalyticsRepository->search($criteria, $salesChannelContext->getContext())->first();
+        $analytics = $this->salesChannelAnalyticsRepository->search($criteria, $salesChannelContext->getContext())->getEntities()->first();
```

Comment:
```suggestion
        $analytics = $this->salesChannelAnalyticsRepository->search($criteria, $salesChannelContext->getContext())->first();
```

the getEntities call should not be necessary anymore to get the correct type :thinking: Didn't you fixed that? :sweat_smile: 
if so, please check the other places, too

Reply:
We can do this without `getEntities()`, yes, but we already discussed this in my proposed PR for the EntitySearchResult changes, where we would remove the helper methods and require the developer to use `getEntities()`.
That's the reason I add it when I see it missing somewhere